otter/cloud_client.py

```diff
@@ -507,6 +524,25 @@ class NovaComputeFaultError(Exception):
     """
 
 
+@attributes([])
```

Comment:
```python
@attr.s
class _BaseExceptionWithMessage(Exception):
     message = attr.ib()
     def __str__(self):
          return self.message

class NovaSomethingSomethingException(_BaseExceptionWithMessage):
     pass
```

Since they all need messages?